src/MatrixDotNet/NetCore/MatrixAsFixedBuffer.cs

```diff
@@ -269,7 +273,10 @@ public static MatrixAsFixedBuffer SubByRef(ref MatrixAsFixedBuffer left, ref Mat
         /// </exception>
         public static MatrixAsFixedBuffer MulByRef(ref MatrixAsFixedBuffer left, ref MatrixAsFixedBuffer right)
         {
-            if (left.Columns != right.Rows) throw new MatrixDotNetException("");
+            if (left.Columns != right.Rows)
+            {
+                throw new MatrixDotNetException("");
```

Comment:
May be exception message should be more informative?)

Reply:
ooooooooops)